### PR TITLE
documenting the sitemap features

### LINKED ISSUE #681: document SeoBundle sitemap configuration and extending
document the sitemap configuration section.

document tagging custom sitemap loader, voter and guessers.

code changes see https://github.com/symfony-cmf/SeoBundle/pull/225

### PR DESCRIPTION
**Pull request by dbu**
documenting https://github.com/symfony-cmf/SeoBundle/pull/225 assuming that https://github.com/symfony-cmf/SeoBundle/pull/239 is also merged.

fix #681

### CODE REVIEW

**Comment by dbu** [comment]
@ElectricMaxxx may i ask you to review this one as well?

File: bundles/seo/configuration.rst [hunk 1]
```diff
@@ -122,6 +122,98 @@ manually or disable the content listener.
     In versions of the SeoBundle prior to 1.2, the ``content_key`` was
     configured directly in the ``cmf_seo`` root.
 
+``sitemap``
+~~~~~~~~~~~
+
+.. versionadded:: 1.3
+    Support for sitemaps was added in version 1.3 of the SeoBundle.
```
Comment (WouterJ) [inline bundles/seo/configuration.rst]: "was introduced"

File: bundles/seo/configuration.rst [hunk 2]
```diff
@@ -122,6 +122,98 @@ manually or disable the content listener.
...
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+
```
Comment (WouterJ) [inline bundles/seo/configuration.rst]: this line should be removed

File: bundles/seo/sitemap.rst [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+=================
+
+This bundle helps loading documents that should go in a sitemap, extracting
+the information from them and rendering them to a sitemap information.
+
+You can generate sitemaps in different formats and it is possible to provide
+more than one set of configurations.
+
+.. versionadded:: 1.3
+    Sitemap support has been added in version 1.3.
```
Comment (WouterJ) [inline bundles/seo/sitemap.rst]: "Sitemap support was introduced in SeoBundle 1.3."

File: bundles/seo/sitemap.rst [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+
+The minimal configuration is to just enable sitemap support with the default
+settings:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+
```
Comment (WouterJ) [inline bundles/seo/sitemap.rst]: this line has to be removed

File: bundles/seo/sitemap.rst [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+provided by the bundle for that format.
+
+The templates are specified in the ``defaults`` section or for a specific sitemap.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+
```
Comment (WouterJ) [inline bundles/seo/sitemap.rst]: same here

File: bundles/seo/configuration.rst (hunk 1 above)
Comment (ElectricMaxxx) [inline bundles/seo/configuration.rst]: I think the next Version for seo bundle is 1.2 isn't it?

Comment (ElectricMaxxx) [inline bundles/seo/configuration.rst]: yep our last version was 1.1.

**Comment by ElectricMaxxx** [comment]
Except the version thing, this documentation looks very good and explains the complecity of the configuration in a very short way. Great.

File: bundles/seo/configuration.rst (hunk 1 above)
Comment (dbu) [inline bundles/seo/configuration.rst]: oups, indeed!

**Comment by dbu** [comment]
updated, thanks for the reviews

**Comment by ElectricMaxxx** [comment]
@dbu this is your friendly reminder service: please be aware of the changes made by https://github.com/symfony-cmf/SeoBundle/pull/243 thanks.

**Comment by dbu** [comment]
updated for default sitemaps

**Comment by dbu** [comment]
@ElectricMaxxx ok if we merge this and then document the features we are now cleaning up in separate PRs? would be more manageable imho.

**Comment by ElectricMaxxx** [comment]
:+1: would be nice if this is merged, an i can use the reference on my
PR :-)
Am 31.07.2015 um 15:55 schrieb David Buchmann:

**Pull request merged**